x/stake/keeper/validator.go

```diff
@@ -663,7 +651,7 @@ func (k Keeper) bondValidator(ctx sdk.Context, validator types.Validator) types.
 
 	// add to accumulated changes for tendermint
 	bzABCI := k.cdc.MustMarshalBinary(validator.ABCIValidator())
-	store.Set(GetTendermintUpdatesKey(validator.OperatorAddr), bzABCI)
+	store.Set(GetTendermintUpdatesTKey(validator.OperatorAddr), bzABCI)
```

Comment:
Should be `tstore.Set()` where `tstore` is `ctx.TransientStore(k.storeTKey)`

Reply:
👍 thank you!